RevenueCatUI/Views/GradientView.swift

```diff
@@ -80,23 +80,23 @@ struct GradientView: View {
     }
 
     var body: some View {
-        switch gradientStyle {
-        case .linear(let degrees):
-            GeometryReader { geometry in
+        GeometryReader { geometry in
+            switch gradientStyle {
+            case .linear(let degrees):
                 let points = calculatePoints(angle: .degrees(Double(degrees)), rect: geometry.frame(in: .local))
                 LinearGradient(
                     gradient: gradient,
                     startPoint: points.start,
                     endPoint: points.end
                 )
+            case .radial:
+                RadialGradient(
+                    gradient: gradient,
+                    center: .center,
+                    startRadius: 0,
+                    endRadius: min(geometry.size.width, geometry.size.height)
```

Comment:
Unlike with color stops, the start and end radii are point values and not percentages. So 100 didn't push to the bounds of the view, it pushed  it to 100 points then it was whatever the last color of the gradient was